src/components/Clothing.js

```diff
@@ -1,11 +1,69 @@
-import styled from 'styled-components/macro'
+import { useState } from 'react'
+import styled, { css } from 'styled-components/macro'
 
-function Clothing({ imageUrl }) {
-  return <StyledImage src={imageUrl} alt="" />
+function Clothing({ titel, color, pattern, material, fitting, imageUrl }) {
+  const [showDetails, setShowDetails] = useState(false)
+
+  return (
+    <StyledSection>
+      <StyledButton
+        showDetails={showDetails}
+        onClick={() => {
+          if (showDetails) {
+            setShowDetails(false)
+          } else {
+            setShowDetails(true)
+          }
+        }}
+      >
+        <StyledImage src={imageUrl} alt="" />
+        {showDetails ? (
+          <StyledWrapper>
+            <h2>{titel}</h2>
+            <p>Farbe: {color}</p>
+            <p>Muster: {pattern}</p>
+            <p>Material: {material}</p>
+            <p>Passform: {fitting}</p>
+          </StyledWrapper>
+        ) : (
+          <div></div>
+        )}
+      </StyledButton>
+    </StyledSection>
+  )
 }
 
+const StyledSection = styled.section`
+  overflow: ${({ showDetails }) => (showDetails ? 'hidden' : 'auto')};
+`
+
+const StyledButton = styled.button`
+  border: none;
+  background: white;
+
+  ${({ showDetails }) =>
+    showDetails &&
+    css`
+      width: 100vw;
+      height: 100vh;
+      position: fixed;
+      z-index: 1;
+      left: 0;
+      top: 0;
+    `}
+`
+
 const StyledImage = styled.img`
-  width: 200px;
+  width: 100%;
+  border: solid 0.1px rgba(0, 0, 0, 0.2);
+  border-radius: 10px;
+  box-shadow: 2px 2px 2px 1px rgba(0, 0, 0, 0.2);
```

Comment:
for reoccuring values you could create global variables in the `GlobalStyles.js` like so:
```
: root {
  --default-border: solid 0.1px rgba( 0,0,0,0.2);
  --default-shadow: 2px 2px 2px 1px rgba(0,0,0,0.2);
}
```

and then use them in the styles:
```
  width: 100%;
  border: var(--default-border);
  border-radius: 10px;
  box-shadow: var(--default-shadow);
```
That way you don't have to update every single value manually in the future.